examples/nidigital/configure-voltage-levels-and-termination-voltage.py

```diff
@@ -0,0 +1,200 @@
+
+     load_pattern_response = nidigital_client.LoadPattern(nidigital_types.LoadPatternRequest(vi = vi,
+         file_path = os.path.join(dir, "Pattern.digipat")))
+     CheckForError(vi, load_pattern_response.status)
+
+     configure_voltage_response = nidigital_client.ConfigureVoltageLevels(nidigital_types.ConfigureVoltageLevelsRequest(vi = vi,
+         channel_list = channelList,
+         vil = vil,
+         vih = vih,
+         vol = vol,
+         voh = voh,
+         vterm = vterm))
+     CheckForError(vi, configure_voltage_response.status)
+     
+     if termSelect == 'h':
+        configure_termination_Mode_response = nidigital_client.ConfigureTerminationMode(nidigital_types.ConfigureTerminationModeRequest(vi = vi,
+            channel_list = channelList,
+            mode = nidigital_types.TerminationMode.TERMINATION_MODE_NIDIGITAL_VAL_HIGH_Z))
+        CheckForError(vi, configure_termination_Mode_response.status)
+
+     if termSelect == 'a':
+        configure_termination_Mode_response = nidigital_client.ConfigureTerminationMode(nidigital_types.ConfigureTerminationModeRequest(vi = vi,
+            channel_list = channelList,
+            mode = nidigital_types.TerminationMode.TERMINATION_MODE_NIDIGITAL_VAL_ACTIVE_LOAD))
+        CheckForError(vi, configure_termination_Mode_response.status)
+
+        
+        configure_active_load_levels_response = nidigital_client.ConfigureActiveLoadLevels(nidigital_types.ConfigureActiveLoadLevelsRequest(vi = vi,
+            channel_list = channelList,
+            iol = iol,
+            ioh = ioh,
+            vcom = vcom))
+        CheckForError(vi, configure_active_load_levels_response.status)
+
+     if termSelect == 't':
+        configure_termination_mode_response = nidigital_client.ConfigureTerminationMode(nidigital_types.ConfigureTerminationModeRequest(vi = vi,
+            channel_list = channelList,
+            mode = nidigital_types.TerminationMode.TERMINATION_MODE_NIDIGITAL_VAL_VTERM))
+        CheckForError(vi, configure_termination_mode_response.status)
+
+     if(termSelect != 'h' and termSelect != 'a' and termSelect != 't'):
+        print('The value of termSelect is invalid. Please set it to one of \'h\', \'a\', or \'t\' values.')
+        exit(0)
+     
+     burst_pattern_response = nidigital_client.BurstPattern(nidigital_types.BurstPatternRequest(vi = vi,
+        site_list = "",
+        start_label = "new_pattern",
+        select_digital_function = True,
+        wait_until_done = True,
+        timeout = 10))
+     CheckForError(vi, burst_pattern_response.status)
+
+     select_function_response = nidigital_client.SelectFunction(nidigital_types.SelectFunctionRequest(vi = vi,
+        channel_list = channelList,
+        function_raw = nidigital_types.SelectedFunction.SELECTED_FUNCTION_NIDIGITAL_VAL_DISCONNECT))
+     CheckForError(vi, select_function_response.status)
+
+     print('NI-Digital Pattern Device is configured without error.\n')
```

Comment:
Use one of these two wordings (note the capitalization of "device"):
- NI-Digital Pattern device configured successfully.
- The NI-Digital Pattern device was configured successfully.